cartoframes/examples.py

```diff
@@ -0,0 +1,381 @@
+"""Download, preview, and query example datasets for use in cartoframes
+examples. Try examples by `running the notebooks in binder
+<https://mybinder.org/v2/gh/CartoDB/cartoframes/master?filepath=examples>`__,
+or downloading them from the `cartoframes examples directory
+<https://github.com/CartoDB/cartoframes/tree/master/examples>`__.
+
+In addition to the functions listed below, this examples module provides a
+:py:class:`CartoContext <cartoframes.context.CartoContext>` that is
+authenticated against all public datasets in the https://cartoframes.carto.com
+account. This means that besides reading the datasets from CARTO, users can
+also create maps from these datasets.
+
+For example, the following will produce an interactive map of poverty rates in
+Census tracts in Brooklyn, New York (preview of static version below code).
+
+    .. code::
+
+        from cartoframes.examples import example_context
```

Comment:
N00b people like me love those examples, copy-and-paste friendly, that include `import` lines in every snippet :heart: